<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TemperatureUnitSettingsPane</class>
 <widget class="QWidget" name="TemperatureUnitSettingsPane">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>76</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QRadioButton" name="systemRadioButton">
     <property name="text">
      <string>Auto</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="celsiusRadioButton">
     <property name="text">
      <string>Celsius</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="fahrenheitRadioButton">
     <property name="text">
      <string>Fahrenheit</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
